arm-sql/2014-04-01/swagger/importExport.json

```diff
@@ -0,0 +1,495 @@
+                        "schema": {
+                            "$ref": "#/definitions/ImportExportOperationStatusResponse"
+                        }
+                    }
+                },
+                "x-ms-long-running-operation": true
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/export": {
+            "post": {
+                "tags": [
+                    "ImportExport"
+                ],
+                "operationId": "Databases_Export",
+                "description": "Exports a Azure SQL Database to a bacpac.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    },
+                    {
+                        "name": "databaseName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL database to be exported."
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ExportRequestParameters"
+                        },
+                        "description": "The required parameters for exporting a database."
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ImportExportOperationStatusResponse"
+                        }
+                    },
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ImportExportOperationStatusResponse"
+                        }
+                    },
+                    "201": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ImportExportOperationStatusResponse"
+                        }
+                    }
+                },
+                "x-ms-long-running-operation": true
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/importExportOperationResults/{operationId}": {
```

Comment:
There is no API for listing the operation IDs.  The operation ID needs to be obtained from the response of initiating the operation.

---
In reply to: [80283654](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80283654) [](ancestors = 80283654)